index.js

```diff
@@ -1,10 +1,12 @@
 import fp from 'fastify-plugin'
 import graphql from 'graphql'
+import { DEFAULT_OPTIONS } from './lib/constant.js'
 
-export default fp(async fastify => {
+export default fp(async (fastify, userOptions) => {
+  const options = { ...DEFAULT_OPTIONS, ...userOptions }
+  if (!options.enabled) return
```

Comment:
This is a temporary solution we need to rethink the access to the data. This won't be achieved enabling/disabling the plugin.